File: Javascript/Algorithms/771.js
/**
 * @param {string} J
 * @param {string} S
 * @return {number}
 */
let numJewelsInStones = (J, S) => {
    const h = new Set(J);
    let count = 0;
    for (let c of S){
        if (h.has(c)){
            count += 1;
        }
    }
    return count;
}; 

File: Rust/Algorithms/771.rs
use std::collections::HashSet;

impl Solution {
    pub fn num_jewels_in_stones(j: String, s: String) -> i32 {
        let h: HashSet<_> = j.chars().collect();
        let mut count = 0;
        for c in s.chars() {
            if h.contains(&c){
                count += 1;
            }
        }
        count
    }
}
